README.md

```diff
@@ -23,7 +23,7 @@ Available CDNs are:
 <body>
 <form name="checkoutForm" method="POST" action="test.html">
     <!--Sample checkout for a 995.00 THB charge-->
-    <script type="text/javascript" src="https://cdn.omise.co/card.js.gz"
```

Comment:
dont worry, I have some small changes to add, I will fix it.
Also, what about https://cdn.omise.co/card-2.0.0.js ? I see it's not present in CDN.